main/src/com/google/refine/exporters/CsvExporter.java

```diff
@@ -97,9 +98,19 @@ public void export(Project project, Properties params, Engine engine, final Writ
             (params != null && params.getProperty("printColumnHeader") != null) ?
                 Boolean.parseBoolean(params.getProperty("printColumnHeader")) :
                 true;
-        
-        final CSVWriter csvWriter = 
-            new CSVWriter(writer, separator.charAt(0), CSVWriter.DEFAULT_QUOTE_CHARACTER, lineSeparator);
+
+        final CSVWriter csvWriter;
+        if(separator.charAt(0) == '\t'){
+            csvWriter = new CSVWriter(writer, separator.charAt(0), CSVWriter.DEFAULT_QUOTE_CHARACTER, CSVWriter.NO_ESCAPE_CHARACTER);
```

Comment:
What happens if one of the cells of the project contains a "\t" character? It would be good to have a test case to demonstrate the behaviour there.

Reply:
Hi,
I tried recreating the issue with a "\t"  character in the cell, however, it does not split the cell. I believe \t is taken care of by the Quote character.

With regard to the test case, I think exportTsvWithComma in TsvExporterTests.java has a test for a tab instead of a comma, Copied below is the snippet for ready reference

    @Test
    public void exportTsvWithComma() {
        CreateGrid(3, 3);

        project.rows.get(1).cells.set(1, new Cell("with\t tab", null));
        try {
            SUT.export(project, options, engine, writer);
        } catch (IOException e) {
            Assert.fail();
        }

        Assert.assertEquals(writer.toString(), "column0\tcolumn1\tcolumn2\n" +
                "row0cell0\trow0cell1\trow0cell2\n" +
                "row1cell0\t\"with\t tab\"\trow1cell2\n" +
                "row2cell0\trow2cell1\trow2cell2\n");
    }
   
   
  Please correct me if I have not understood anything, and please guide me on a further course of action for this 
  issue. I will be highly obliged for the same.

Reply:
@wetneb.

Reply:
ok thanks, this case seems fine to me - but I am not sure about another one, let me comment there.